docs/notebooks/01_merge.ipynb

```diff
@@ -1,27 +1,489 @@
 {
```

Comment:
<p>maybe add short description why you are manually assigning those. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>